tracee/tracee.go

```diff
@@ -423,6 +429,11 @@ func (t *Tracee) populateBPFMaps() error {
 		fileFilterMap.Update(uint32(i), []byte(t.config.FilterFileWrite[i]))
 	}
 
+	uidHash, _ := t.bpfModule.GetMap("uid_filter")
+	for i := 0; i < len(t.config.Filter.UIDs); i++ {
+		uidHash.Update(uint32(t.config.Filter.UIDs[i]), uint32(1))
```

Comment:
Just an idea, tell me what you think
The value of the uid_map can take one of, say, "0", "1", "2", which will represent - "<", "=", ">"
Then the user will be able to specify uid filters as:
"--filter uid>0", for example

Anyways, we don't have to implement this now

Reply:
I like it! Can also have one that represents "not" (like uid != 1000).

I can add that as part of this PR or in a follow up.

Reply:
If it's not too many additions, you can add it here,
otherwise, we can do a follow up PR

Reply:
I'll give it a shot and you can make the call on if you want me to cherry-pick the changes onto a new PR instead. I typically like small PRs.